theano/gpuarray/opt.py

```diff
@@ -310,7 +338,14 @@ def apply(self, fgraph):
             for lopt in (self.local_optimizers_all +
                          self.local_optimizers_map.get(type(node.op), []) +
                          self.local_optimizers_map.get(node.op, [])):
-
+                
+                nb = change_tracker.nb_imported
+                process_count[lopt] += 1
+                global_process_count[lopt] += 1
+                t_opt = time.time()
+                lopt_change = self.process_node(fgraph, node, lopt)
```

Comment:
Do not call process_node. This won't do that we want. So you won't need a change_tracker.